### PR TITLE
Adding Persian to Whitelist locales

### PR DESCRIPTION
**Pull request by BaseMax**
Hello,

Where we can watch alpha version?
https://opensourcefriday.com/?locale=fa

Target: https://github.com/github/opensourcefriday/pull/571
- Adding to available locales
- Adding to languages.yml -  `fa: Persian (فارسی)`

Best,
Max Base
Open Source Maintainer on GitHub

### CODE REVIEW

File: config/languages.yml [hunk 1]
```diff
@@ -44,9 +44,10 @@ default: &default
   es: Español
   et: Eesti
   eu: Euskara
+  fa: Persian (فارسی)
```
Comment (MikeMcQuaid) [inline config/languages.yml]:
```suggestion
  fa: فارسی
```

File: config/languages.yml [hunk 2]
```diff
@@ -44,9 +44,10 @@ default: &default
...
+  fj: Na Vosa Vakaviti
   ff: Fulfulde
   fi: Suomi
-  fj: Na Vosa Vakaviti
```
Comment (MikeMcQuaid) [inline config/languages.yml]: Please restore this line to how it was, thanks!

File: config/languages.yml (hunk 1 above)
Comment (BaseMax) [reply config/languages.yml]: Solved.

File: config/languages.yml (hunk 2 above)
Comment (MikeMcQuaid) [reply config/languages.yml]: @BaseMax Still this one to do, thanks!

Comment (BaseMax) [reply config/languages.yml]: I think solved. 👌😬

**Comment by MikeMcQuaid** [comment]
Thanks again @BaseMax!

**Comment by BaseMax** [comment]
Thank you 🙏👌👌

Where we can watch alpha version?
https://opensourcefriday.com/?locale=fa

**Comment by MikeMcQuaid** [comment]
> Where we can watch alpha version?

Sorry, I don't understand the question?

> https://opensourcefriday.com/?locale=fa

That link is live and working 🎉

**Pull request merged**